packages/core/docs/general/where-to-start.md

```diff
@@ -0,0 +1,106 @@
+
+At the start, you should familiarize yourself with our **Guides** section. Includes all that you need to know for successful shop implementation. Also, we have a section with the most advanced topics.
+
+**Guides** section will guide you through:
+- [Theme](../guide/theme.html)
+- [Configuration](../guide/configuration.html)
+- [Composables](../guide/composables.html)
+- [Getters](../guide/getters.html)
+- [Product catalog](../guide/product-catalog.html)
+- [Authentication](../guide/authentication.html)
+- [User profile](../guide/user-profile.html)
+- [Cart and Wishlist](../guide/cart-and-wishlist.html)
+- [Checkout](../guide/checkout.html)
+
+**Advanced** section will guide you through:
+- [Architecture](../advanced/architecture.html)
+- [Application context](../advanced/context.html)
+- [Calling platform API](../advanced/calling-platform-api.html)
+- [Extending GraphQL Queries](../advanced/extending-graphql-queries.html)
+- [Server Middleware](../advanced/server-middleware.html)
+- [Internationalization](../advanced/internationalization.html)
+- [Performance](../advanced/performance.html)
+- [SSR Cache](../advanced/ssr-cache.html)
+- [Logging](../advanced/logging.html)
+- [API Reference](../core/api-reference/)
+
+To be up to date, we recommend to following [Migration guide](../migrate) and Vue Storefront 2 [Roadmap](https://www.notion.so/vuestorefront/Vue-Storefront-2-Next-High-level-Roadmap-201cf06abb314b84ad01b7b8463c0437) .
+
+## Integrator
+
+If you are using a custom backend or want to use platforms or technologies not listed on our [Integrations](../integrations/) page, you likely need to create an integration.
+
+Vue Storefront integrates with a wide range of technologies and each has its own needs, so there is no generic tutorial for all of them.
+
+However, depending on the kind of integrations you want to build, you should familiarize yourself with **Building Integration** section. Especially integration guide for [eCommerce](../integrate/integration-guide.html) and [CMS](../integrate/cms.html) platforms.
+
+**eCommerce** section will guide you through: 
+- [Introduction](../integrate/integration-guide.html#introduction)
+- [Scope](../integrate/integration-guide.html#scope)
+- [Getting started](../integrate/integration-guide.html#getting-started)
+- [Creating an api-client](../integrate/integration-guide.html#creating-an-api-client)
+- [Creating composables](../integrate/integration-guide.html#creating-composables)
+- [Creating getters](../integrate/integration-guide.html#creating-getters)
+- [Creating a theme](../integrate/integration-guide.html#creating-a-theme)
+
+**CMS** section will guide you through:
+- [configuration / content fetching / content rendering](../integrate/cms.html#what-is-needed)
+- [usage example in the real application](../integrate/cms.html#usage-example-in-the-real-application)
```

Comment:
Remove these lists. IMHO the links above are enough. We don't need to list each section.